File: src/dotnet/ArdsTools/Ards.NetCore.Adif.Domain/Models/Award.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Ards.NetCore.Adif.Domain.Models
{
    [Table("award", Schema = "adif")]
    public partial class Award
    {
        [Column("id")]
        public int Id { get; set; }

        [Column("name")]
        [StringLength(15)]
        public string Name { get; set; }

        [Required]
        [Column("import_only")]
        public bool? ImportOnly { get; set; }
    }
}

File: src/java/ArdsAdifApi/src/main/java/com/ards/adif/api/model/Award.java
package com.ards.adif.api.model;

import java.io.Serializable;
import javax.persistence.*;


/**
 * The persistent class for the award database table.
 * 
 */
@Entity
@Table(name="award")
@NamedQuery(name="Award.findAll", query="SELECT a FROM Award a")
public class Award implements Serializable {
	private static final long serialVersionUID = 1L;

	@Id
	@Column(unique=true, nullable=false)
	private Integer id;

	@Column(name="import_only")
	private Boolean importOnly;

	@Column(nullable=false, length=15)
	private String name;

	public Award() {
	}

	public Integer getId() {
		return this.id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public Boolean getImportOnly() {
		return this.importOnly;
	}

	public void setImportOnly(Boolean importOnly) {
		this.importOnly = importOnly;
	}

	public String getName() {
		return this.name;
	}

	public void setName(String name) {
		this.name = name;
	}

}